src/Command/Core/CoreConfigureCommand.php

```diff
@@ -104,130 +86,152 @@ protected function execute(InputInterface $input, OutputInterface $output)
+                'Adapters displayed with a "<info>*</info>" are already configured.',
+                'You are recommended to not skip this step if you configure Gush for the first time.'
+            ]
+        );
+        $styleHelper->newLine();
 
-        if (null === $issueTrackerName && null === $input->getOption('adapter')) {
-            $selection = array_search($adapterName, array_keys($issueTrackers), true);
+        // Run in a loop to allow multiple selection
+        while (true) {
+            $adapterName = $styleHelper->numericChoice('Choose adapter', $labels);
 
-            if (false === $selection) {
-                $selection = null;
+            if ('noop' === $adapterName) {
+                break;
             }
 
-            $issueTrackerName = $styleHelper->askQuestion(
-                new ChoiceQuestion('Choose issue tracker: ', array_keys($issueTrackers), $selection)
-            );
-        } elseif (null !== $issueTrackerName && !array_key_exists($issueTrackerName, $issueTrackers)) {
-            throw new UserException(
-                sprintf(
-                    'The issue tracker "%s" is invalid. Available adapters are "%s"',
-                    $issueTrackerName,
-                    implode('", "', array_keys($issueTrackers))
-                )
-            );
-        }
+            $this->configureAdapter($input, $output, $adapterName, $adapters[$adapterName]);
+            $this->handleDefaulting($adapterName, $adapters[$adapterName]);
 
-        if (null !== $issueTrackerName) {
-            $this->configureAdapter($input, $output, $issueTrackerName, 'issue_trackers');
-
-            $currentDefault = $this->config->get('issue_tracker');
-            if ($issueTrackerName !== $currentDefault &&
-                $styleHelper->askQuestion(
-                    new ConfirmationQuestion(
-                        sprintf('Would you like to make "%s" the default issue tracker?', $issueTrackerName),
-                        null === $currentDefault
-                    )
-                )
-            ) {
-                $this->config->merge(['issue_tracker' => $issueTrackerName]);
+            if (!$styleHelper->confirm('Do you want configure other adapters?', false)) {
+                break;
             }
         }
 
-        $cacheDir = $styleHelper->askQuestion(
-            (new Question(
-                "Cache folder [{$this->config->get('cache-dir')}]: ",
-                $this->config->get('cache-dir')
-            ))->setValidator(
-                function ($dir) {
-                    if (!is_dir($dir)) {
-                        throw new \InvalidArgumentException('Cache folder does not exist.');
-                    }
-
-                    if (!is_writable($dir)) {
-                        throw new \InvalidArgumentException('Cache folder is not writable.');
-                    }
+        $styleHelper->section('VersionEye configuration');
```

Comment:
:+1: for removing VersionEye, for anyone how really wants this we can always look into moving it to its own project (separated from Gush).

The only thing VersionEye needs is a token, Gush is more about Repository management and Issue-tracking.